pipeline/metadata/5.2.yaml

```diff
@@ -0,0 +1,152 @@
+#=====================================================================================
+# Metadata file for 5.2 RHCS release version.
+# Single file to execute the all tier-1, tier-2, tier-3 and tier-4 suites in parallel
+# as per defined in each stages.
+# Each stage will execute in sequential pattern.
+# Contains all Default parameter used for Execution when no override is specified.
+# parameter required to create a similar custom file is suite name, suite yaml file, global configuration file,
+# platform, rhbuild, inventory and metadata information like frequency of execution, tier, cloud type, functional group and stage.
+#=====================================================================================
+- name: "Deploy 5x cluster and apply services through cli"
+  suite: "suites/pacific/cephadm/tier-0_5-1_cephadm.yaml"
+  global-conf: "conf/pacific/cephadm/sanity-cephadm.yaml"
+  platform: "rhel-8"
+  rhbuild: "5.2"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - dmfg
+    - stage-1
+
+- name: "Deploy all services at 5x using a spec file"
+  suite: "suites/pacific/cephadm/test-container-cli-args.yaml"
+  global-conf: "conf/pacific/cephadm/test-container-cli-args.yaml"
+  platform: "rhel-8"
+  rhbuild: "5.2"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - dmfg
+    - stage-1
+
+- name: "Upgrade cluster from 5x GA to 5x latest"
+  suite: "suites/pacific/upgrades/tier-1_upgrade_cephadm.yaml"
+  global-conf: "conf/pacific/upgrades/tier-1_upgrade_cephadm.yaml"
+  platform: "rhel-8"
+  rhbuild: "5.2"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - upgrades
```

Comment:
Please add dmfg also as a tag along with upgrades. Follow this for all upgrade suites.